packages/utils/src/fallbackSubgraph.ts

```diff
@@ -313,10 +405,97 @@ export class FallbackSubgraph<T extends SubgraphSdk> {
     // Always start with the in sync subgraphs and then concat the out of sync subgraphs.
     // Metrics should only come in to play to sort subgraph call order within each group (i.e. we should never prioritize
     // an out-of-sync subgraph over a synced one).
-    return this.subgraphs
+    return subgraphs
       .filter((subgraph) => subgraph.record.synced)
       .sort((subgraphA, subgraphB) => subgraphA.priority - subgraphB.priority)
-      .concat(this.subgraphs.filter((subgraph) => !subgraph.record.synced))
+      .concat(subgraphs.filter((subgraph) => !subgraph.record.synced))
       .sort((subgraphA, subgraphB) => subgraphA.priority - subgraphB.priority);
   }
+
+  /**
+   * Utility for creating a subgraph record and filling it out with the default values.
+   * Does not add it to this.subgraphs.
+   *
+   * @param url - The url of the subgraph.
+   * @returns Subgraph<T> where the generic type T represents the client SDK type.
+   */
+  private createSubgraphRecord(url: string): Subgraph<T> {
+    const getSubgraphName = (url: string) => {
+      const split = url.split("/");
+      return split[split.length - 1];
+    };
+    return {
+      url,
+      client: this.generateClient(url),
+      record: {
+        name: getSubgraphName(url),
+        synced: true,
+        latestBlock: -1,
+        syncedBlock: -1,
+        lag: 0,
+        error: undefined,
+      },
+      priority: 0,
+      metrics: {
+        calls: [],
+        cps: 0,
+        reliability: 0,
+        avgExecTime: 0,
+      },
+    };
+  }
 }
+
+// TODO: Remove, replace with API call / hosted data.
+export const getDeployedAnalyticsSubgraphUrls = (chainId: number, chainData?: Map<string, ChainData>): string[] => {
```

Comment:
Any particular reason for the function name change? Otherwise breaks interfaces unnecessarily

Reply:
why? who's calling this? They shouldn't be - and this should be removed. This hardcoded stuff is intended to be deprecated when there's a health endpoint for analytics as well.

Reply:
I think it's fine to leave the same names for the functions since these are external. Let's not make assumptions about who is calling what. This function should not necessarily be deprecated, it should use chainData instead if it can.